### DISCUSSION TITLE
Is it possible to use listener pattern with Laravel CRUD?

### DISCUSSION THREAD

**Discussion by angeloadd** [orchidsoftware/platform#2139] [Q&A] [answered]
Hi everybody and thanks in advance for the help.
I have a Select with multiple branding option and I want to display options from an other select based on an option selected on the first one.
From Documentation the suggested method is to use Listeners with Screens but I am using Resources not Screens to display data. Is it possible to implement this pattern in Laravel CRUD and Resources?
There are other ways to obtain this behavior apart from using custom javascript?

**Comment by tabuna** [comment, marked as answer]
Hey @AngeloAdd The CRUD package has a straightforward use. Therefore, it lacks many of the available features on the platform. To use the listener, you need to create a screen.

**Reply by angeloadd** [reply]
Thank you for the answer 🙏🏻
